Dockerfile

```diff
@@ -1,7 +1,8 @@
 #--------- Generic stuff all our Dockerfiles should start with so we get caching ------------
+ARG DISTRO=debian
 ARG IMAGE_VERSION=buster
 ARG IMAGE_VARIANT=-slim
-FROM debian:$IMAGE_VERSION$IMAGE_VARIANT
+FROM $DISTRO:$IMAGE_VERSION$IMAGE_VARIANT
```

Comment:
Can you also add a small section in the README with an example of how to build using the args

Reply:
Added.

Reply:
Thanks . Can you fix the failing test by changing the sleep time to 30 in https://github.com/kartoza/docker-postgis/blob/develop/scenario_tests/datadir_init/test.sh

Reply:
done, but maybe there's a better way than hardcoded sleep. https://github.com/jwilder/dockerize is a small golang application that can handle waiting for dbs with timeouts etc

Reply:
thanks, We only use the sleep in unit testing. Maybe you could add a PR for this we would be glad to review it